ludwig/schema/combiners/common_transformer_options.py

```diff
@@ -52,9 +52,17 @@ class CommonTransformerConfig:
         parameter_metadata=COMBINER_METADATA["TransformerCombiner"]["use_bias"],
     )
 
-    bias_initializer: Union[str, Dict] = common_fields.BiasInitializerField()
+    bias_initializer: Union[str, Dict] = schema_utils.InitializerOrDict(
+        default="zeros",
```

Comment:
Not sure I understand the goal of these changes. The entire purpose for introducing the `BiasInitializerField` was to:

1. Standardize the description of all bias initializers to include all options, give an informative description, etc.
2. Standardize the default params.
3. Standardize the parameter metadata.

This change seems to be a step backward by reverting those changes, but I'm not sure what the benefit is?

Reply:
Thanks the call out.

My main goal here is to standardize more of the config against a consistent set of schema structures for our initialization parameters to prime the schema for initializer parameter schemafication, not realizing that `Bias/Weights/EmbeddingInitializerField` was more evolved than `InitializerOrDict`.

I'll standardize on `Bias/Weights/EmbeddingInitializerField` then, for its better descriptions and parameter metadata.